pages/index.js

```diff
@@ -1,7 +1,93 @@
+import Image from "next/image";
+import styled from "styled-components";
+import icon from "../src/cocktail_icon.svg"
+import cocktail from "../src/old_fashioned.jpeg"
+
+
+
+console.log(icon);
+
 export default function Home() {
   return (
     <div>
-
+      <Headlinebox>
+        <Headline>Cocktail App</Headline>
+        <div>
+          <Image
+            src={icon}
+            alt="menu icon"
+            width={70}
+            height={70}
+          />
+        </div>
+      </Headlinebox>
+      <Cocktail>
+        <Image
+          src={cocktail}
+          alt="menu icon"
+        />
+        <Cocktailname>
+          Old Fashioned
+        </Cocktailname>
+      </Cocktail>
+      <Search>
+        <form>
+          <label for="quicksearch" ></label>
+          <input type="text" id="quicksearch" />
```

Comment:
So far you didn't add any functionality to your search-field. We want only commit features that have a functionality, so you could still leave out this input field until you also want to add functionality :)